From: bolor2022 <759085099@qq.com>
Date: Thu, 13 Nov 2025 11:08:48 +0000
Subject: Translated using Weblate (Mongolian)

Currently translated at 89.5% (188 of 210 strings)

Translation: openkylin-nile-sp2/kylin-os-manager-device-controller
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-sp2/kylin-os-manager-device-controller/mn/
---
 .../translations/kylin-os-manager-device-controller_mn.ts               | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/device-controller/translations/kylin-os-manager-device-controller_mn.ts b/plugins/device-controller/translations/kylin-os-manager-device-controller_mn.ts
index 06a3f02..472910a 100644
--- a/plugins/device-controller/translations/kylin-os-manager-device-controller_mn.ts
+++ b/plugins/device-controller/translations/kylin-os-manager-device-controller_mn.ts
@@ -761,7 +761,7 @@
         <location filename="../src/include/hwinfostorage.h" line="30"/>
         <location filename="../src/include/hwinfostorage.h" line="45"/>
         <source>CPU Temperature</source>
-        <translation>CPU ᠳᠤᠯᠠᠭᠠᠨ ᠤ ᠬᠡᠮᠵᠢᠶ᠎ᠡ ᠃</translation>
+        <translation>CPU ᠳᠤᠯᠠᠭᠠᠨ ᠤ᠋ ᠬᠡᠮᠵᠢᠶ᠎ᠡ</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="49"/>
